README.md

```diff
@@ -108,9 +107,10 @@ _\*Especially in today's times it's hard to always be politically correct. Addit
 
 Nothing that would break something but some small things could be observed:
 
-- When vscode's "smooth scrolling" setting is enabled and persistent scrolling (pressing and holding the key) is executed, there can be small inconsistencies during scrolling. The reason for this is that it is not possible to implement async await functionality for many of the commands accessible through vscode's API.
-- When placing the cursor between folded sections and using a scroll command with a numeric value, small scrolling inconsistencies can occasionally occur.
-- When scrolling in VisualMode beyond the currently visible area, the selection becomes invisible (it reappears when you move the cursor after scrolling).
+- Scrolling inconsistencies may occur:
+   - During fast scrolling (e.g., pressing and holding the key) with the "smooth scrolling" setting enabled.
+   - During scrolling actions between folded sections.
+- Selection becomes invisible when scrolling in VisualMode beyond the currently visible range (it reappears when moving the cursor after scrolling).
```

Comment:
@tobealive these changes look great, does this section suggest turning off smooth scrolling?

Reply:
also have you considered trying to get your work merged into vscode vim? I think it would benefit a lot more people

Reply:
Hey @roryfahy,

Yep I have mentioned it after the initial release in one or two spots.
It was great to see that other users recommended it's users in vscode/neovim as we have similar issues with scrolling there. 

The comment about smooth scrolling is mainly mentioning the hardship of getting it perfect.
I recommend turning it on anyway, as this visual assistance helps the eye follow along. Takes less brain power to process then.